Product Management Service/Code/product_management_service.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE products (
+  id int PRIMARY KEY,
+  name varchar(255),
+  price decimal(10,2),
+  category varchar(255),
```

Comment:
Do we need to maintain this as a [ENUM](https://dev.mysql.com/doc/refman/8.0/en/enum.html)? Or shall we be handling this in the business layer?

Reply:
This can be handled in business layer. We can provide a few basic broad categories in the business logic. This will ensure ease of maintainability in the database as we don't have to keep updating the ENUM field's values there and maintain versioning.

Reply:
So, basically, we will give some broader categories and not give option to add/remove categories.